nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml

```diff
@@ -628,8 +634,13 @@
                                 js/d3/**/*,
                                 js/codemirror/**/*,
                                 js/jquery/**/*,
+                                js/**/**/*,
```

Comment:
Why was this line added? It appears to be includes all the JS resources that was aggregated above.